packages/common/src/utils/regex/RegexUtils.tsx

```diff
@@ -28,4 +28,19 @@ export const RegexUtils = {
     const matcher = new RegExp(substring, 'i');
     return matcher.test(testString);
   },
+  // Case-insensitive version of 'startsWith'
+  matchString: (substring: string, testString: string) => {
+    const matcher = new RegExp(`^${substring}`, 'i');
+    return matcher.test(testString);
+  },
+  // matches test string against an object's name or code properties
+  matchNameOrCode: (
```

Comment:
mSupply generally only searches using startsWith, so I've kept that tradition.. although I find the `includes` more helpful myself